test/e2e/credentials/stack.go

```diff
@@ -216,19 +214,19 @@ func (s *e2eCfnStack) createInstanceProfile(ctx context.Context, logger logr.Log
 	return instanceProfileArn, nil
 }
 
-func (s *e2eCfnStack) instanceProfileName(roleName string) string {
+func (s *Stack) instanceProfileName(roleName string) string {
 	return roleName
 }
 
-func (e *e2eCfnStack) readStackOutput(ctx context.Context, logger logr.Logger) (*e2eCfnStackOutput, error) {
-	resp, err := e.cfn.DescribeStacksWithContext(ctx, &cloudformation.DescribeStacksInput{
-		StackName: aws.String(e.stackName),
+func (e *Stack) readStackOutput(ctx context.Context, logger logr.Logger) (*StackOutput, error) {
```

Comment:
Now that we have removed e2e from those struct's name, do we want to rename this variable `e` to something else?